### PR TITLE
mgr: Miscellaneous small mgr fixes

### PR DESCRIPTION
**Pull request by jcsp**
Just a few small bits spun off while working on other things

### CODE REVIEW

File: src/mgr/ActivePyModules.cc [hunk 1]
```diff
@@ -374,17 +374,18 @@ int ActivePyModules::start_one(PyModuleRef py_module)
 
   assert(modules.count(py_module->get_name()) == 0);
 
+  // Thread names must be <16 chars
+  std::string short_name = py_module->get_name().substr(0, 15);
```
Comment (tchaikov) [inline src/mgr/ActivePyModules.cc]: could remove this local variable, as it not referenced after being defined.

File: src/mgr/PyModuleRunner.h [hunk 2]
```diff
@@ -46,12 +46,18 @@ class PyModuleRunner
...
   };
 
+  std::string thread_name;
 
 public:
   int serve();
   void shutdown();
   void log(int level, const std::string &record);
 
+  const char *get_thread_name()
```
Comment (tchaikov) [inline src/mgr/PyModuleRunner.h]: could mark this method `const` .

File: src/mgr/DaemonServer.cc [hunk 3]
```diff
@@ -708,7 +708,11 @@ bool DaemonServer::handle_command(MCommand *m)
         con->mark_disposable();
       }
 
-      dout(1) << "handle_command " << cpp_strerror(r) << " " << rs << dendl;
+      if (r == 0) {
+        dout(4) << "handle_command success" << dendl;
```
Comment (tchaikov) [inline src/mgr/DaemonServer.cc]: nit, use `__func__` instead.

**Comment by jcsp** [comment]
Thanks for comments, pushed an update to fix them.

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2018-07-08_02:34:43-rados-wip-kefu-testing-2018-07-07-0953-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2018-07-09_06:54:15-rados-wip-kefu-testing-2018-07-07-0953-distro-basic-mira/

test failures not caused by environment issues are tracked by http://tracker.ceph.com/issues/24640

**Pull request merged**